app/lib/client/client.dart

```diff
@@ -32,14 +32,24 @@ class SPHclient {
   String password = "";
   String schoolID = "";
   String schoolName = "";
-  Map<SPHAppEnum, LoadApp>? applets;
+
+  /// The list of fetchers and which are used by the different views.
+  /// Properties of each LoadApp can change.
+  final Map<SPHAppEnum, LoadApp> applets = {};
+
+  /// Loaded from [loadFromStorage] to be *only* used in [initialiseApplets].
+  /// Map of bools which indicate if a applet should be periodically loaded.
+  /// Just a convenience variable because of how [initState] in [startup.dart] works.
+  late final Map<String, dynamic> shouldFetchApplets;
```

Comment:
Du schreibst map of bools, dann mach dynamic zu bool

Reply:
Wie gesagt, wenn man es zu bool ändert macht es Probleme. Die App bleibt im Ladebildschirm hängen. Das liegt daran weil es in loadFromStorage() mit jsonDecode() assignt wird und das gibt Map<String, dynamic> zurück. `as Map<String, bool>` bringt auch nichts.